README.md

```
@@ -1,101 +1,112 @@
+
 For Software:
+
 # Installation
+
 [commands]
 
 # Run
+
 [commands]
 
 ### Project Documentation
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)